spec/graphql/resolvers/current_user_resolver_spec.rb

```diff
@@ -28,6 +31,24 @@
     end
   end
 
+  describe 'with revoked membership' do
+    let(:membership) { create(:membership) }
+    let(:revoked_membership) do
+      create(:membership, user: membership.user, status: :revoked)
+    end
+
+    before { revoked_membership }
+
+    it 'only lists organizations when membership has an active status' do
+      result = execute_graphql(
+        current_user: membership.user,
+        query: query,
+      )
+
+      expect(result['data']['currentUser']['organizations'].length).to eq(1)
```

Comment:
Can you check that revoked membership is not present instead?

Something like:
`expect(result['data']['currentUser']['organizations']).not_to include(...)`

Reply:
I've changed with this code, ok for you ? 

`expect(result['data']['currentUser']['organizations']).not_to include(revoked_membership.organization)`